### PR TITLE
Release Machine Stats main using tags

### PR DESCRIPTION
**Pull request by pranavmalaviya**
This PR removes the previous approach of using `bump2version` and replaces it with tags. With this, you won't need a dev setup to release a new machine stats version.

To make a new release, simply create a tag on the `master` branch and the `pypi-upload` GitHub Workflow will take care of the rest.

### CODE REVIEW

File: unix/setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="machine_stats",
-    version="2.5.4",
+    version="develop",
```
Comment (jmariomejiap) [inline unix/setup.py:8]: Im curious about this as well. can you please explain to us why the change to develop?

File: .github/workflows/pypi-upload.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ name: PyPI Upload
 
 on:
   push:
```
Comment (jmariomejiap) [inline .github/workflows/pypi-upload.yml:4]: I was under the impression that we were aiming to use `releases :`. Is that change coming soon? or this is just a leftover?

File: .github/workflows/pypi-upload.yml [hunk 3]
```diff
@@ -16,8 +17,23 @@ jobs:
...
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Autobump version
+        run: |
+          # from refs/tags/v1.2.3 get 1.2.3
+          VERSION=${GITHUB_REF/refs\/tags\//}
+          PLACEHOLDER='version="develop"'
```
Comment (jmariomejiap) [inline .github/workflows/pypi-upload.yml:29]: I see the develop line here. 🤔

File: libvirt/CONTRIBUTING.md [hunk 4]
```diff
@@ -83,6 +83,12 @@ We use the following tools:
        1    debian10   running
       ```
 
+### How to bump libvirt's version
```
[the remark this answers is not in the document]
Comment (pranavmalaviya) [reply libvirt/CONTRIBUTING.md:86]: Yes, checkout https://pypi.org/project/virt-stats/
It's been under the radar because it is released with the same version as machine stats (because it's the same workflow). So you'll see a lot of virtstat versions without any changes (and yet a new version) because those versions were only intended for machine stats.

File: .github/workflows/pypi-upload.yml (hunk 3 above)
Comment (pranavmalaviya) [reply .github/workflows/pypi-upload.yml:29]: To also answer your other 2 questions about `develop`, it can be any string (or even be empty).
The magic in this workflow is actually happening in line#36. After stripping off the variables, it'll look something like

```
sed -i "s/develop/1.2.3/g" script.py
```

So we are replacing the string `develop` in that file with the tag(1.2.3) that we get from the GitHub context. This also means that we have no way to know what's the latest MS version from codebase without having to look at GitHub or PyPI.

Comment (jmariomejiap) [reply .github/workflows/pypi-upload.yml:29]: Cool, thanks for the explanation 👍

File: .github/workflows/pypi-upload.yml (hunk 2 above)
Comment (pranavmalaviya) [reply .github/workflows/pypi-upload.yml:4]: A change is coming and I wanted to keep it the same as what we have in alpha but first I'm checking in a private repo to make sure the tags don't get mixed up, meaning releasing a tag for alpha shouldn't release the main machine stats.

**Review by jmariomejiap — APPROVED** [review]
Thanks @pranavmalaviya 🚀
